src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -280,7 +287,7 @@ export default class StyleConfigurator {
       if (bpmnElement instanceof ShapeBpmnEvent) {
         styleValues.set(StyleIdentifier.BPMN_STYLE_EVENT_KIND, bpmnElement.eventKind);
 
-        if (bpmnElement instanceof ShapeBpmnBoundaryEvent) {
+        if (bpmnElement instanceof ShapeBpmnBoundaryEvent || bpmnElement instanceof ShapeBpmnStartEvent) {
```

Comment:
:warning: missing test for this new feature (I didn't notice it during my 1st review)

Reply:
yes, good catch